src/components/PrimaryPanes/Outline.css

```diff
@@ -72,6 +77,10 @@
   white-space: nowrap;
 }
 
+.outline-list > .outline-list__element {
+  padding-left: 0;
```

Comment:
So, instead of removing the list item's padding, and letting the icon's padding do the actual spacing, I would remove the padding-left on the icon, then add more padding to `.outline-list__element` when it's in a sub-list.

Reply:
Ok, changes made:

Outline.css:43 - added padding when element is on a class list
Outline.css:90 -- remove padding-left from icon